gstreamer/src/caps.rs

```diff
@@ -397,6 +397,67 @@ impl Builder {
     }
 }
 
+#[cfg(feature = "ser_de")]
+pub(crate) mod serde {
```

Comment:
Maybe we can move those serde submodules into different files, like `src/caps_serde.rs`, and then just import them into `src/caps.rs`? One level of indentation less and it's clearer where to find the code

Reply:
Indeed. Some of them are getting quite large too.